spec.md

```diff
@@ -238,31 +238,43 @@ following order of precedence:
 There is no merging of information from different sources. Only the
 source with the highest precedence is considered.
 
-## Format of copyright notices
+## Format of Copyright Notices
+
+A Copyright Notice MUST start with a tag, word or symbol (collectively:
+prefixes) from the following list:
 
-A copyright notice MUST be prefixed by a tag, symbol or word denoting a
-copyright notice as described in this specification.
+- `SPDX-FileCopyrightText` (or `SPDX-SnippetCopyrightText` in Snippets)
+- Copyright
+- ©
 
-The copyright notice MUST contain the name of the copyright holder. The
+It is RECOMMENDED to use the `SPDX-FileCopyrightText` tag. You MAY add '(C)' or
+'©' after the prefix.
```

Comment:
missing '(c)' (lowercase)

Reply:
@silverhook Do you have a good half-sentence rationale that we could add at the end of the quoted sentence? Something like 'You MAY choose to add [...] after the prefix in order to $very_short_rationale_here'.

Reply:
How about:

> You MAY choose to add [...] after the prefix _to have the best of both worlds – have the copyright notice easy to find using the SPDX tag as well as following the form established by the Berne convention_.

Reply:
I'm going to close that as being too long and opinionated.